client/my-sites/hosting/php-version-card/index.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import Spinner from 'components/spinner';
+import FormRadiosBar from 'components/forms/form-radios-bar';
+import { errorNotice, successNotice } from 'state/notices/actions';
+
+const requestId = ( siteId, method ) => `hosting-php-version-${ method }-${ siteId }`;
+
+export const requestPhpVersion = siteId => {
+	const method = 'GET';
+
+	return requestHttpData(
+		requestId( siteId, method ),
+		http(
+			{
+				method,
```

Comment:
Minor, there's only one usage here so 'GET' would be just fine

Reply:
`requestId` is generated using `siteId` and `method`.